### PR TITLE
Takeaway- Challenge

### PR DESCRIPTION
**Pull request by CazaBelle**

### CODE REVIEW

File: spec/order_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      expect(order.print_basket).to eq("You have ordered pizza, pasta and the total is 19.95")
+    end 
+
+  describe '#complete_order' do
+    it 'will send a text' do 
+      expect(order.complete_order).to eq "Text sent"
+    end 
+  end
+  end
+  end
```
Comment (stonefarmer9) [inline spec/order_spec.rb]: Why have you used so many describe blocks?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,43 @@
-Takeaway Challenge
+
+Takeaway Challenge - Caz's Calzones
```
Comment (joaoag) [inline README.md]: Yes. 
This name is excellent, really excellent. 💯 💯 💯

File: README.md [hunk 3]
```diff
@@ -1,34 +1,43 @@
...
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
-
- ```
 
-Instructions
+Description
```
Comment (joaoag) [inline README.md]: Very clear instructions and installation guidelines 👍

File: README.md [hunk 4]
```diff
@@ -48,32 +57,16 @@ So that I am reassured that my order will be delivered on time
...
+Built using `Twilio-Ruby`
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+Tests
+-----
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+run `rspec` in the command line
```
Comment (joaoag) [inline README.md]: Ooh yes instructions on how to run the tests, hadn't thought of doing that, nice one.

File: lib/menu.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+class Menu
+  attr_reader :dishes 
+  def initialize 
+    @dishes = [{ name: "pizza", price: 10.50 }, 
+      { name: "pasta", price: 9.45 },
+      { name: "calzone", price: 8.65 },
```
Comment (joaoag) [inline lib/menu.rb]: Only one Calzone at Caz's Calzones? Not going to lie I am a bit disappointed :' (

**Pull request closed without merging**